content/applications/hr/recruitment/recruitment_analysis.rst

```diff
@@ -0,0 +1,103 @@
+====================
+Recruitment analysis
+====================
+
+The *Recruitment Analysis* report allows recruiting departments to see which job positions have
+the most applicants, which have the most referrals, and how long it takes for applicants to move
+through the pipeline.
+
+Knowing how many applicants each specific job position has, along with statistics about how many are
+hired and refused, can provide valuable insights. This information can assist the recruiting team to
+pivot their strategies to acquire the most desirable candidates.
+
+Recruitment analysis report
+===========================
+
+Start by navigating to :menuselection:`Recruitment app --> Reporting --> Recruitment Analysis`.
+This presents a line chart of all applicants for the last year.
+
+Three separate color-coded metrics are presented: :guilabel:`In Progress`, :guilabel:`Hired`, and
+:guilabel:`Refused`.
+
+Hover the cursor over a month of the chart and a pop-up window appears, displaying the specific
+numbers for that month.
+
+.. image:: recruitment_analysis/line-chart.png
+   :align: center
+   :alt: The default Recruitment Analysis report.
+
+Pivot table view
+----------------
+
+For a more detailed view of the information in the :guilabel:`Recruitment Analysis` report, click
+the :icon:`oi-view-pivot` :guilabel:`(Pivot)` icon in the top-right corner. This displays all the
+information in a pivot table.
+
+In this view, the job positions are displayed in the rows, and the columns display the total numbers
+of applicants, including how many of those applicants were hired or refused. The displayed
+information can be modified, if desired.
+
+In this example, there are 17 total applicants. Out of that 17, three have been hired, and four
+refused. Each job position lists the corresponding number of applicants hired and refused. The
```

Comment:
```suggestion
refused. The
```
opting to remove this sentence because this same information was just represented in the paragraph above